src/ratchet.ts

```diff
@@ -44,7 +44,7 @@ function ratchetSingleNumberCoverage(
       ? Math.round(category.pct) - options.ratchetPercentagePadding
       : category.pct;
     if (num > 0 && num <= ratchetPct) {
-      return ratchetPct;
+      return options.roundPct ? Math.floor(ratchetPct) : ratchetPct;
```

Comment:
If this is rounding, shouldn't this use `Math.round`?  How would you feel about using `options.floorPct`?  Since rounding and flooring are two different things.

Reply:
Thanks for the quick response! That's a very good point. I'll rename it to `options.floorPct`.